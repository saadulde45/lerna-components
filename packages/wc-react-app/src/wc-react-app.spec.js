import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import WCReactApp from './wc-react-app';

describe('WC React App Tests', () => {
    it('should render correctly', () => {
        const wrapper = mount(
            <WCReactApp />
        );

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
