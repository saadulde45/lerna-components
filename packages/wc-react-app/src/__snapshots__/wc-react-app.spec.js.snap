// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WC React App Tests should render correctly 1`] = `
<WCReactApp>
  <div
    className="parent-wrapper"
  >
    <span
      className="title"
    >
      React Child Updated
    </span>
  </div>
</WCReactApp>
`;
